packages/larva-patterns/modules/content-showcase-grid/content-showcase-grid.prototype.js

```diff
@@ -0,0 +1,9 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const content_showcase = clonedeep( require( '../../modules/content-showcase/content-showcase.prototype' ) );
+
+module.exports = {
+    content_showcase_classes: 'lrv-u-border-t-1 lrv-u-padding-t-025 lrv-u-font-family-body lrv-a-wrapper-100vw@mobile-max',
+    content_showcase_items_classes: 'content-showcase-items lrv-u-flex lrv-u-flex-direction-column@mobile-max lrv-u-border-t-1 lrv-u-border-b-1 lrv-u-padding-tb-1@desktop lrv-u-width-100p lrv-u-padding-lr-125@mobile-max',  
```

Comment:
These properties should only contain lrv- classes. Any classes that are added based on the name of the module should be coded into the Twig template and should not have CSS associated with them (the purpose of them is to know what module we are looking at when inspecting the DOM).

Reply:
@laras126 I have updated the content showcase module to use o-card so this is fixed now.